addons/website_event/static/src/js/website_event_send_email.js

```diff
@@ -0,0 +1,94 @@
+/** @odoo-module **/
+
+import { rpc } from "@web/core/network/rpc";
+import { checkEmailValidity } from  "@mail/utils/common/format";
+import publicWidget from '@web/legacy/js/public/public_widget';
+
+publicWidget.registry.websiteEventSendEmail = publicWidget.Widget.extend({
+    selector: '.o_wevent_send_by_email_widget',
+    events: {
+        'click .o_wevent_js_send_email': '_onSendEmailClick',
```

Comment:
let us align the naming a bit more : maybe `o_wevent_send_by_email_button` ? `onSendEmailButtonClick` ?